assets/app/views/project.html

```diff
@@ -195,8 +195,9 @@ <h2 ng-if="!displayRouteByService[serviceId]">
           </div>
 
           <!-- TODO implement filters for empty and present to return booleans for cases like this -->
-          <div ng-if="(podsByService[serviceId] | hashSize) == 0" class="osc-object components-panel deployment-block deployments none">
-            <span class="pficon pficon-info"></span> There are currently no pods for this service.
+          <div ng-if="(podsByService[serviceId] | hashSize) === 0 && (deploymentsByServiceByDeploymentConfig[serviceId] | hashSize) === 0"
```

Comment:
Make sure this works when you have an RC that isn't a deployment and it is scaled to zero